thunder/tests/test_networks.py

```diff
@@ -445,6 +445,69 @@ def test_hf_qwen2():
     torch.testing.assert_close(grads_ref, grads_compiled, rtol=1e-2, atol=1e-2)
 
 
+PHI3_MINI_128K_CFG = {
```

Comment:
There's no need to copy the config here because it's not behind registration wall and downloadable with AutoConfig.